test/lib/model/test_yake_keywords.py

```diff
@@ -53,10 +53,10 @@ def test_normalize_special_characters(self):
         expected = "'''\"\""
         self.assertEqual(self.yake_model.normalize_special_characters(text), expected)
 
-    def test_run_chinese_segmentation_with_jieba(self):
+    def test_run_chinese_segmentation(self):
         test_text = '''哈里斯同意与特朗普再进行一次美大选辩论'''
-        expected = "哈里 里斯 哈里斯 同意 与 特朗普 再 进行 一次 美 大选 辩论"
-        self.assertEqual(self.yake_model.run_chinese_segmentation_with_jieba(test_text), expected)
+        expected = "哈里斯 同意 与 特朗 普再 进行 一 次 美 大选 辩论"
```

Comment:
can we add tests with some of the examples I provided where jieba was giving multiple terms?